### ISSUE TITLE
Possible issues with stat command in concurrent use cases

### ISSUE THREAD

**Issue by ieugen** [juji-io/datalevin#91]
Using datalevin 0.5.27 I run a job to copy and compress some xml values from one datalevin db to another datalevin db.

During copying I run a few times
 dtlv -d test2.lmdb -a stat
and I manage to crash the app doing the copying with this.
It happens consistently.
Is this expected?

> Huahai  3:11 AM
> Probably the stat command is not implemented properly for concurrent use case like this, could you file an issue? thanks.

```
cat /etc/debian_version 
bookworm/sid

apt-cache show liblmdb0 
Package: liblmdb0
Source: lmdb
Version: 0.9.24-1
Installed-Size: 104
I can share jvm dump probably but erhaps privatly
WARNING: All illegal access operations will be denied in a future release
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGBUS (0x7) at pc=0x00007f325b9ddd8d, pid=489175, tid=489277
#
# JRE version: OpenJDK Runtime Environment 18.9 (11.0.12+7) (build 11.0.12+7)
# Java VM: OpenJDK 64-Bit Server VM 18.9 (11.0.12+7, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [lmdbjava-native-library-12939233817828088453.so+0x5d8d]  mdb_page_alloc.isra.19+0x44d
#
# Core dump will be written. Default location: Core dumps may be processed with "/lib/systemd/systemd-coredump %P %u %g %s %t 9223372036854775808 %h" (or dumping to workspace/core.489175)
```

**Comment by huahaiy** [comment]
Acknowledged

**Comment by huahaiy** [comment]
This should be resolved when we are done with #184.
